api-server/apiserver.py

```diff
@@ -1,13 +1,46 @@
 import os
 from functools import partial
 
+from datadog import initialize as datadog_initialize, statsd
+
 import sentry_sdk
 from flask import Flask, json, abort, jsonify, request
 from semver import VersionInfo
 from sentry_sdk.integrations.flask import FlaskIntegration
 from werkzeug.contrib.cache import SimpleCache
 
 
+class Metrics:
+
+    PREFIX = os.getenv("METRICS_PREFIX", "release_registry")
+
+    def initialize(self, **kwargs):
+        # DATADOG_API_KEY, DATADOG_APP_KEY can be provided from env
+        # TODO(michal): Pass statsd_constant_tags from env?
+        datadog_initialize(**kwargs)
+        return self
+
+    @staticmethod
+    def tags_from_reqeust():
+        tags = []
+        if request.url_rule:
+            tags.append(f"route:{request.url_rule.rule}")
+        return tags
+
+    def _metric(self, name):
+        return f"{self.PREFIX}.{name}"
+
+    def increment(self, name, value=1, tags=None, sample_rate=None):
+        statsd.increment(
+            self._metric(name),
+            value=value,
+            tags=self.tags_from_reqeust() + (tags or []),
+            sample_rate=sample_rate,
+        )
+
+
+metrics = Metrics().initialize()
```

Comment:
Just to make sure, this will not make the app fail if DATADOG_* or any other parameters are not present? 
Also, would be nice to make statsd endpoint configurable.

Reply:
> Just to make sure, this will not make the app fail if DATADOG_* or any other parameters are not present?
Nope, that's safe.

> Also, would be nice to make statsd endpoint configurable.
It is configurable using the DD_AGENT_HOST and DD_DOGSTATSD_PORT env vars (https://github.com/DataDog/datadogpy/blob/master/datadog/dogstatsd/base.py#L64-L70)